google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java

```diff
@@ -501,36 +502,26 @@ private Builder(EnhancedBigtableStubSettings settings) {
      * constructing the callable chain.
      */
     private static class PlaceholderBatchingDescriptor
```

Comment:
I don't think the placeholder should be necessary anymore

Reply:
Removed the same.